lib/src/tasks/saucelabs/sauce_iframe_runner/sauce_iframe_runner.dart

```diff
@@ -300,8 +300,8 @@ Future<List<String>> runTests(String testUrl, String platform,
 
   // Load the tests
 
-  var augmentedTestUrl = new Fluri(testUrl)
-    ..setQueryParam('platform', platform);
+  var augmentedTestUrl = new UriBuilder.fromUri(Uri.parse(testUrl))
+    ..queryParameters = {'platform': platform};
```

Comment:
There's no need for a uri package at all here, the builtin `Uri` class will suffice:

```dart
var augmentedTestUrl = Uri.parse(testUrl).replace(queryParameters: {'platform': platform});
```